pwnlib/libcdb.py

```diff
@@ -71,13 +71,15 @@ def provider_libc_rip(hex_encoded_id, hash_type):
     data = b""
     try:
         result = requests.post(url, json=params, timeout=20)
-        if result.status_code != 200 or len(result.json()) == 0:
+        result.raise_for_status()
+        if len(result.json()) == 0:
```

Comment:
We might only do this once.
```suggestion
        libc_match = result.json()
        if not libc_match:
```